### PR TITLE
[Documentation] Data types for Dictionary exclude & sort column

### PR DESCRIPTION
**Pull request by sgururajshetty**
The supported datatype mentioned for dictionary exclude and sort columns

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

File: docs/data-management-on-carbondata.md [hunk 1]
```diff
@@ -44,13 +44,16 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
   
    - **Dictionary Encoding Configuration**
 
-     Dictionary encoding is turned off for all columns by default from 1.3 onwards, you can use this command for including columns to do dictionary encoding.
+     Dictionary encoding is turned off for all columns by default from 1.3 onwards, you can use this command for including or excluding columns to do dictionary encoding.
      Suggested use cases : do dictionary encoding for low cardinality columns, it might help to improve data compression ratio and performance.
 
      ```
      TBLPROPERTIES ('DICTIONARY_INCLUDE'='column1, column2')
+	 TBLPROPERTIES ('DICTIONARY_EXCLUDE'='column1, column2')
```
Comment (chenliang613) [inline docs/data-management-on-carbondata.md]: By default, all columns are DICTIONARY_EXCLUDE, no need to specify.

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by manishgupta88** [comment]
LGTM

**Pull request closed without merging**